Doc/c-api/intro.rst

```diff
@@ -119,7 +119,8 @@ complete listing.
 
 .. c:macro:: Py_ABS(x)
 
-   Return the absolute value of ``x``.
+   Return the absolute value of ``x``.  If the result cannot be represented in
+   same type, the behavior is undefined.
```

Comment:
I would like to add an example ("for example, INT_MIN if x is int").

Also, maybe write this as a separate paragraph?